scripts/WinInstallFiles/InstallJMRI.nsi

```diff
@@ -1106,28 +1141,70 @@ Function CheckJRE
   JRESearch:
     IntOp $JREINSTALLCOUNT $JREINSTALLCOUNT + 1
     ClearErrors
-    ReadRegStr $1 HKLM "SOFTWARE\JavaSoft\JRE" "CurrentVersion"
-    ReadRegStr $0 HKLM "SOFTWARE\JavaSoft\JRE\$1" "JavaHome"
-    IfErrors 0 JRECheck
-    ReadRegStr $1 HKLM "SOFTWARE\JavaSoft\Java Runtime Environment" "CurrentVersion"
-    ReadRegStr $0 HKLM "SOFTWARE\JavaSoft\Java Runtime Environment\$1" "JavaHome"
-    IfErrors 0 JRECheck
-    ReadRegStr $1 HKLM "SOFTWARE\JavaSoft\JDK" "CurrentVersion"
-    ReadRegStr $0 HKLM "SOFTWARE\JavaSoft\JDK\$1" "JavaHome"
-    IfErrors 0 JRECheck
-    ReadRegStr $1 HKLM "SOFTWARE\JavaSoft\Java Development Kit" "CurrentVersion"
-    ReadRegStr $0 HKLM "SOFTWARE\JavaSoft\Java Development Kit\$1" "JavaHome"
-
-    ; -- Not found
-    IfErrors 0 JRECheck
-      ; -- If we've got an error here on x64, switch to the 32-bit registry,
-      ; -- decrease the counter and then re-try
-      StrCmp 0 $x64JRE JREInitInstall
-        SetRegView 32
-        DetailPrint "Setting x86 registry view..."
-        StrCpy $x64JRE 0
-        IntOp $JREINSTALLCOUNT $JREINSTALLCOUNT - 1
-        Goto JRESearch
+  ;--check directory for jre
+  ;--Dont think we can check this d
```

Comment:
Is "d" short for directory? If so, it might be good to write directory instead of d, since it makes it easier for others to read.

Reply:
Done